synapse/http/server.py

```diff
@@ -942,10 +953,17 @@ def set_clickjacking_protection_headers(request: Request) -> None:
     request.setHeader(b"Content-Security-Policy", b"frame-ancestors 'none';")
 
 
-def respond_with_redirect(request: Request, url: bytes) -> None:
-    """Write a 302 response to the request, if it is still alive."""
+def respond_with_redirect(
+    request: Request, url: bytes, statusCode: int = FOUND, cors: bool = False
```

Comment:
hypocritical again but please add the new argument to the docstring :innocent:

Reply:
I've documented all the args. 👌